diesel_compile_tests/tests/compile-fail/empty_insertable_struct.rs

```diff
@@ -0,0 +1,20 @@
+#[macro_use]
+extern crate diesel;
+#[macro_use]
+extern crate diesel_codegen;
+
+use diesel::*;
+
+table! {
+    users {
+        id -> Integer,
+    }
+}
+
+#[derive(Insertable)]
+//~^ ERROR no rules expected the token `]`
```

Comment:
Is this the best error we get? No "cannot derive Insertable for struct with no fields"? doesn't matter, should be a new issue in that case